docs/src/user_docs/stdlib/math/u64.md

```diff
@@ -0,0 +1,69 @@
+| unchecked_neq |  Performs inequality comparison of two unsigned 64 bit integers.<br /> The input values are assumed to be represented using 32 bit limbs, but this is not checked.<br /> Stack transition looks as follows:<br /> [b_hi, b_lo, a_hi, a_lo, ...] -> [c, ...], where c = 1 when a != b, and 0 otherwise. |
+| checked_neq |  Performs inequality comparison of two unsigned 64 bit integers.<br /> The input values are assumed to be represented using 32 bit limbs, fails if they are not.<br /> Stack transition looks as follows:<br /> [b_hi, b_lo, a_hi, a_lo, ...] -> [c, ...], where c = 1 when a == b, and 0 otherwise. |
+| unchecked_eqz |  Performs comparison to zero of an unsigned 64 bit integer.<br /> The input value is assumed to be represented using 32 bit limbs, but this is not checked.<br /> Stack transition looks as follows:<br /> [a_hi, a_lo, ...] -> [c, ...], where c = 1 when a == 0, and 0 otherwise. |
+| checked_eqz |  Performs comparison to zero of an unsigned 64 bit integer.<br /> The input value is assumed to be represented using 32 bit limbs, fails if it is not.<br /> Stack transition looks as follows:<br /> [a_hi, a_lo, ...] -> [c, ...], where c = 1 when a == 0, and 0 otherwise. |
+| unchecked_min |  Compares two unsigned 64 bit integers and drop the larger one from the stack.<br /> The input values are assumed to be represented using 32 bit limbs, but this is not checked.<br /> Stack transition looks as follows:<br /> [b_hi, b_lo, a_hi, a_lo, ...] -> [c_hi, c_lo, ...], where c = a when a < b, and b otherwise. |
+| checked_min |  Compares two unsigned 64 bit integers and drop the larger one from the stack.<br /> The input values are assumed to be represented using 32 bit limbs, fails if they are not.<br /> Stack transition looks as follows:<br /> [b_hi, b_lo, a_hi, a_lo, ...] -> [c_hi, c_lo, ...], where c = a when a < b, and b otherwise. |
+| unchecked_max |  Compares two unsigned 64 bit integers and drop the smaller one from the stack.<br /> The input values are assumed to be represented using 32 bit limbs, but this is not checked.<br /> Stack transition looks as follows:<br /> [b_hi, b_lo, a_hi, a_lo, ...] -> [c_hi, c_lo, ...], where c = a when a > b, and b otherwise. |
+| checked_max |  Compares two unsigned 64 bit integers and drop the smaller one from the stack.<br /> The input values are assumed to be represented using 32 bit limbs, fails if they are not.<br /> Stack transition looks as follows:<br /> [b_hi, b_lo, a_hi, a_lo, ...] -> [c_hi, c_lo, ...], where c = a when a > b, and b otherwise. |
+
+## Bitwise operations
+| Procedure   | Description   |
+| ----------- | ------------- |
+| checked_and |  Performs divmod operation of two unsigned 64 bit integers.<br /> The input values are assumed to be represented using 32 bit limbs, but this is not checked.<br /> Stack transition looks as follows:<br /> [b_hi, b_lo, a_hi, a_lo, ...] -> [r_hi, r_lo, q_hi, q_lo ...], where r = a % b, q = a / b |
```

Comment:
This description is wrong. It should probably be something like:

"Performs bitwise AND of two unsigned 64-bit integers.
The input values are expected to be represented using 32-bit limbs, and the procedure will fail if they are not.
The tack transition looks as follows:
[b_hi, b_lo, a_hi, a_lo, ...] -> [c_hi, c_lo, ...], where c = a AND b."